beekeeper-package/beekeeper-assembly-scheduler-apiary/pom.xml

```diff
@@ -5,10 +5,10 @@
   <parent>
     <artifactId>beekeeper-package</artifactId>
     <groupId>com.expediagroup</groupId>
-    <version>2.1.1-SNAPSHOT</version>
+    <version>3.0.0-SNAPSHOT</version>
   </parent>
 
-  <artifactId>beekeeper-assembly-path-scheduler-apiary</artifactId>
+  <artifactId>beekeeper-assembly-scheduler-apiary</artifactId>
```

Comment:
I would check with this change that the assembly still builds the docker image successfully. You should be able to run the image locally without any complaints that the java command can't find the app jar

Reply:
Yup seems to be working